internal/prebuiltconfigs/tools/neo4j.yaml

```diff
@@ -0,0 +1,37 @@
+# Copyright 2025 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+sources:
+  neo4j-source:
+    kind: neo4j
+    uri: ${NEO4J_URI}
+    database: ${NEO4J_DATABASE}
+    user: ${NEO4J_USERNAME}
+    password: ${NEO4J_PASSWORD}
+
+tools:
+  execute_cypher:
+    kind: neo4j-execute-cypher
+    source: neo4j-source
+    description: Use this tool to execute Cypher queries.
+
+  schema:
```

Comment:
nit: do you want to add a verb to this tool name? like get_schema or list_schema? It's not needed if testing shows this tool name + description works but would follow the general ecosystem of tool names